### PR TITLE
using Natural Language Understanding

### PR DESCRIPTION
**Pull request by naisofly**
Updated the application to use Watson Natural Language Understanding instead of AlchemyLanguage (which is deprecated)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,84 +1,102 @@
...
+# Sentiment and Emotion Tracker using Natural Language Understanding
 
 Sentiment and Emotion application detect sentiment and emotions from people's digital footprints (e.g., online reviews and social media text) with [IBM Waston](watson) Technology. The application can reveal the overall emotion and sentiment patterns from a text of interest, including the changes and details of positive or negative sentiment and emotions of "anger", "disgust", "sadness", "fear" and "joy".
 
+ ![Sentiment & Emotion Tracker](http://i.imgur.com/kEg5hBi.png)
+
 Give it a try! Click the button below to fork into IBM DevOps Services and deploy your own copy of this application on Bluemix.
 
-[![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/watson-developer-cloud/sentiment-and-emotion)
+[![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/naisofly/sentiment-and-emotion)
```
Comment (germanattanasio) [inline README.md]: update url

File: router/index.js [hunk 2]
```diff
@@ -26,4 +31,28 @@ module.exports = function (app) {
 
   app.use('/api/alchemy-analysis', require('./api/alchemy-analysis'));
 
+
+  app.post('/api/analyze', function(req, res, next) {
+
+    var parameters = {
+      'text' : req.body.text,
```
Comment (germanattanasio) [inline router/index.js]: keys don't need `'`

File: views/banner.jade [hunk 3]
```diff
@@ -4,9 +4,9 @@ div.row.service-container
...
       div.container(onclick="location.href='/'", style="cursor: pointer")
         div.col-lg-10.col-md-10.col-sm-10.col-xs-12.dialog-description
-          h1(style="font-size:50px;").service-title Sentiment and Emotion   
+          h1(style="font-size:50px;").service-title Sentiment and Emotion
           p Sentiment and Emotion application uses text analytics to detect sentiment and emotions from people's digital footprints (e.g., online reviews and social media text). The application can offer positive or negative sentiment and emotions of "anger", "disgust", "sadness", "fear" and "joy".
-          
+
             //- a.resource-item(href='http://www.ibm.com/smarterplanet/us/en/ibmwatson/developercloud/doc/xxx/') Documentation
-            //- a.resource-item(href='https://github.com/watson-developer-cloud/emotion-analysis-nodejs') Fork on Github
+            //- a.resource-item(href='https://github.com/naisofly/sentiment-and-emotion') Fork on Github
```
Comment (germanattanasio) [inline views/banner.jade]: url

File: views/footer.jade [hunk 4]
```diff
@@ -7,6 +7,6 @@ footer.text-center
 		span.separator |
 		a(href='http://www.ibm.com/privacy/us/en/') Privacy Policy
 	div
-		a(href='http://github.com/watson-developer-cloud/sentiment-and-emotion') Fork on Github
+		a(href='https://github.com/naisofly/sentiment-and-emotion') Fork on Github
```
Comment (germanattanasio) [inline views/footer.jade]: url

**Review by germanattanasio — CHANGES_REQUESTED** [review]

**Comment by naisofly** [comment]
Changes committed

File: manifest.yml [hunk 5]
```diff
@@ -1,15 +1,8 @@
 ---
-declared-services:
-  alchemy-service:
-    label: alchemy_api
-    plan: free
 applications:
-- name: sentiment-and-emotion
+- name: nai-emotion
```
Comment (germanattanasio) [inline manifest.yml]: use `sentiment-and-emotion`

File: manifest.yml [hunk 6]
```diff
@@ -1,15 +1,8 @@
...
   command: node app.js
   path: .
   memory: 1G
   services:
-  - alchemy-service
-  env:
-    NODE_ENV: production
-    SECURE_EXPRESS: 1
```
Comment (germanattanasio) [inline manifest.yml]: Why are you removing the environment variables?

File: manifest.yml [hunk 7]
```diff
@@ -1,15 +1,8 @@
...
+  - nlu-test
```
Comment (germanattanasio) [inline manifest.yml]: This is not the name that you use in the readme

File: router/index.js [hunk 8]
```diff
@@ -1,22 +1,27 @@
...
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
 
 'use strict';
-
 module.exports = function (app) {
+  const NaturalLanguageUnderstandingV1 = require('watson-developer-cloud/natural-language-understanding/v1.js');
+  const nlu = new NaturalLanguageUnderstandingV1({
+    'username': process.env.NATURAL_LANGUAGE_UNDERSTANDING_USERNAME,
```
Comment (germanattanasio) [inline router/index.js]: Line 21 and 22 can be commented as the SDK will automatically look at those environment variables

**Review by germanattanasio — CHANGES_REQUESTED** [review]
2nd round

**Comment by naisofly** [comment]
@germanattanasio 2nd round complete

**Comment by naisofly** [comment]
@germanattanasio All check have passed, can we merge this pull request? :smile:

**Review by germanattanasio — APPROVED** [review]

**Pull request merged**